templates/component.ejs

```diff
@@ -11,17 +11,17 @@ import View from '<%= props.appName %>/app/components/base/View';
 const styles = StyleSheet.create({
     container: {},
 });
-
+<% var typeOfComponent = props.isPure ? 'PureComponent' : 'Component' %>
```

Comment:
If you're defining that all the way on top after `// @flow`, you can use `typeOfComponent` in the `import React, { <%= typeOfComponent %> } from 'react';` 😛